README.md

```diff
@@ -150,6 +150,12 @@ This project includes an [Example Instrument](docs/Instrument\ Example.zip) that
 
 ## Acknowledgements & Copyright
```

Comment:
@123andy could you please add attribution for `DERM_PHOTO_BODY` to the end of this section?  
I can handle the preview in the README and instrument.

Reply:
Hey Kyle - I reached out to the person on our team who did that and will add it.  Do you mind merging now as-is so the researcher that has been waiting can move forward and then I'll release a second PR to clean up the issues?

Reply:
Hi Andy, unfortunately I'm not able to actually test at all due to issue #58, which is affecting local instances as well as a staging instance (I wonder if it affects you as well).  
Your logic certainly _looks_ reasonable (aside from the falsy null addressed in the suggestion) but I don't feel confident merging something I can't test at all. @pbchase feel free to override me on this.

Reply:
added - will be in next PR